doc/manual/src/language/constructs.md

```diff
@@ -1,5 +1,372 @@
 # Language Constructs
 
+## Basic Literals
+
+### String {#string-literal}
+
+  *Strings* can be written in three ways.
+
```

Comment:
The content below was moved from `values.md`, with anchors changed to headers.
Did I miss any other changes?

Reply:
I added some TODOs and I may have updated a link or two around here, but the intent was that this step would be a mostly-pure move with the editing to come later.